libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -101,6 +101,24 @@ const swapProviders: Record<string, SwapProviderConfig> = {
   },
   ftx,
   ftxus: ftx,
+  oneinch: {
+    ...createExchangeProviderNameAndSignature({
+      name: "1inch",
+      signature: "",
+      publicKey: "",
+    }),
+    needsKYC: false,
+    needsBearerToken: false,
+  },
+  paraswap: {
+    ...createExchangeProviderNameAndSignature({
+      name: "paraswap",
+      signature: "",
+      publicKey: "",
+    }),
+    needsKYC: false,
+    needsBearerToken: false,
```

Comment:
Can we add in the  if the provider definition a new attribute, type. DEX or a CEX?

Reply:
done [c8822e2](https://github.com/LedgerHQ/ledger-live/pull/2123/commits/c8822e29ecec8e1c7a4c8f34a9d212788fd50272)

Reply:
Not needed